word-guess.rb

```diff
@@ -0,0 +1,156 @@
+# This is a word guessing game (by Erika & Ari)
+# Guess the word before all the petals fall
+# use one class for game functionality
+require 'colorize'
+require 'spicy-proton'
+
+
+class Game
+
+ def initialize(word)
+   @correct_word_array = word.downcase.split("")
+   # This take the random word, downcase, and split letters into it's own indexes
+   # or elements (interchangable right?)
+   @correct_guess = word.gsub(/./ , "_").split("")
+# gsub method lets us sub in "-". "/./" is for any characters.
+# split method puts "-" into own index or makes them their own elements
+   @no_match_guesses = []
+   @failed_attempt = 0
+ end
+
+ def pedal_remove
+   pedal_remove = [
+     '''
+       (@)(@)(@)(@)(@)
+          ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.cyan,'''
+
+        (@)(@)(@)(@)
+        ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.green,'''
+
+          (@)(@)(@)
+         ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.yellow,'''
+
+           (@)(@)
+         ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.blue,'''
+
+              (@)
+          ,\,\,|,/,/,
+             _\|/_
+            |_____|
+             |   |
+             |___|'''.red]
+ end
+
+ def promptUser
+   unless (@correct_guess.include?("_") )
+     puts "\nYOU WIN! THE WORD WAS: #{@correct_word_array.join("")}".green
+     # if user guess all the letters correctly
+      return
+   end
+
+   # ask the user to pick a letter
+   puts "\nGUESS A LETTER!".green
+   guess = gets.chomp.downcase
+
+   # prints guesses to user followed by comma
+   puts "YOUR GUESSES SO FAR: #{@no_match_guesses.join(', ')}"
+
+   # let the user know if they can't have more than one letter
+     if (guess.length > 1)
+       puts "\nYOU HAVE TO GUESS ONE LETTER AT A TIME.".red
+       return promptUser
```

Comment:
This portion, between lines 75 and 91, might be good to break out as a separate method, something like `validate_guess`.